src/site/content/en/learn/html/inline-text/index.md

```diff
@@ -3,10 +3,194 @@ title: 'Other inline text elements'
+The `<em>` element can be nested, with each level of nesting indicating a greater degree of emphasis. This element has semantic
+meaning and can be used to inform auditory user agents like screen readers, Alexa, and Siri, to provide emphasis.
+* Use the [`<mark>`](https://developer.mozilla.org/docs/Web/HTML/Element/mark) element to identify or highlight text
+that is somehow relevant, like highlighting (or "marking") the occurrence of search terms in search results. This enables
+marked content to be quickly identified without adding emphasis or importance.
+* The [`<strong>`](https://developer.mozilla.org/docs/Web/HTML/Element/strong) element identifies text as having strong
+importance. Browsers usually  render the content with a heavier font weight.
+* The [`<cite>`](https://developer.mozilla.org/docs/Web/HTML/Element/cite) element, covered in [text basics](/learn/html/text-basics/#quotes-and-citations),
+is used to mark the titles of books, articles, or other creative work, or an abbreviated reference or citation metadata for such,
+like a book's ISBN number.
+
+## Deprecated elements
+
+There are three elements that were temporarily deprecated, but have been added back into HTML. They should be used sparingly,
+if at all, as they provide little to no semantic value and CSS should always be used for styling over HTML elements.
+
+### `<i>`
+
+The [`<i>`](https://developer.mozilla.org/docs/Web/HTML/Element/i) elements can be used for technical terms, foreign words
+(again, with the [`lang`](https://developer.mozilla.org/docs/Web/HTML/Global_attributes/lang) attribute identifying the
+language), thoughts, or ship names. The element is used to differentiate inline content from the surrounding text for a specific reason,
+like idiomatic text, technical terms, and taxonomic designations. This element should not be used simply to italicize text.
+
+MLW uses a `<span>` element for the weird text at the bottom of Toasty McToastface's workshop review. The [`<span>`](https://developer.mozilla.org/docs/Web/HTML/Element/span)
+element provides for a generic inline container that has no semantics and doesn't represent anything. This would have also been an appropriate use of `<i>`.
+
+{% Codepen {
+user: 'web-dot-dev',
+id: 'mdGbrBd'
+} %}
+
+The default style for the `<i>` element is to render the element in italic font. In this example, we set `font-style: normal`
+because the characters used are not available in italic.
+
+### `<u>`
+
+The [`<u>`](https://developer.mozilla.org/docs/Web/HTML/Element/u) element is for that has non-textual annotation. For example,
+you may want to annotate knowingly misspelled words. By default, the content is underlined, but this can be controlled with CSS,
+such as by adding a red wavy underline to mimic word processor grammar error indicators.
+
+<p>I always spell <u>licence</u> wrong</p>
```

Comment:
```suggestion
```html
<p>I always spell <u>licence</u> wrong</p>
\```
```

this should be code; not sure how to edit this in github without ending it early